modules/howtos/examples/Search.java

```diff
@@ -40,6 +40,98 @@
 import reactor.core.publisher.BaseSubscriber;
 import reactor.core.publisher.Mono;
 
+// This example assumes an index called `travel-sample-index` exists, 
+// you can create it by running the curl command below:
```

Comment:
It's good to list prereqs, but this is a very long curl command, also repeated in another java file in this commit, so at some point, this will all go out of sync.

Is it better at this point to point the reader at the init-buckets.sh script instead?

(Or, is the N1QL statement to create the index any shorter?)

Reply:
hmm, I think probably pointing to init-buckets.sh would be easier 👍